.editorconfig

```diff
@@ -5,6 +5,10 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
 indent_size = 2
+end_of_line = lf
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Yeah, this is necessary, otherwise it's hard to have an appropriate line break in markdown 😅 

@takezoe Thoughts?